R/plumber.R

```diff
@@ -902,7 +902,13 @@ plumber <- R6Class(
     },
     #' @description Retrieve openAPI file
     openAPIFile = function() {
-      self$swaggerFile()
+      warning("`$openAPIFile()` has been deprecated in v1.0.0 and will be removed in a coming release. Please use `$apiSpec()`.")
```

Comment:
Consider using lifecycle (if you anticipate wanting to avoid repeated warnings) and/or `.Deprecated`

Reply:
If so, would be good in another PR (this is non-blocking)